src/main/scala/us/diempham/taskman/database/InMemoryTaskDatabase.scala

```diff
@@ -0,0 +1,13 @@
+package us.diempham.taskman.database
+
+import us.diempham.taskman.application.TaskStorage.{Task, TaskId}
+import us.diempham.taskman.application.UserStorage.UserId
+
+class InMemoryTaskDatabase extends InMemoryDatabase[TaskId, Task] with TaskDatabaseInterface {
+  def getTasksForUser(userId: UserId): List[Task] = {
+    database.filter { case (_, task) => task.userId == userId}.toList.map(_._2)
+  }
+
+  override def create(k: TaskId, v: Task): Task = super.create(k, v)
```

Comment:
This create method is not necessary, since all it is doing is calling the inherited method.